cognite/client/_api/data_modeling/instances.py

```diff
@@ -555,7 +555,7 @@ def search(
         instance_type: Literal["node", "edge"] = "node",
         properties: list[str] | None = None,
         filter: Filter | dict | None = None,
-        limit: int = INSTANCES_LIST_LIMIT_DEFAULT,
+        limit: int = 1000,
```

Comment:
```suggestion
        limit: int = LIST_LIMIT_DEFAULT,
```
Suggest doing the same here, with same argument as above.